docs/examples.md

```diff
@@ -228,3 +228,87 @@ cleanup() {
 setup
 #cleanup
 ```
+## Rancher for administering the cluster
+
+Install rancher in to the same cluser for administering the cluster.
+
+```
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cattle-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rancher
+  namespace: cattle-system
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: rancher
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+# This role binding allows "jane" to read pods in the "default" namespace.
```

Comment:
I think, the names changed slightly :grin:

Reply:
lol. Yeah, will do that